E0453 needs to be updated to new format

From: src/test/compile-fail/E0453.rs

E0453 needs a span_label and a span_note changed to a span_label, updating it from:

```
error[E0453]: allow(non_snake_case) overruled by outer forbid(non_snake_case)
  --> src/test/compile-fail/E0453.rs:13:9
   |
13 | #[allow(non_snake_case)] //~ ERROR E0453
   |         ^^^^^^^^^^^^^^
   |
note: `forbid` lint level set here
  --> src/test/compile-fail/E0453.rs:11:11
   |
11 | #![forbid(non_snake_case)]
   |           ^^^^^^^^^^^^^^
```

To:

```
error[E0453]: allow(non_snake_case) overruled by outer forbid(non_snake_case)
  --> src/test/compile-fail/E0453.rs:13:9
   |
11 | #![forbid(non_snake_case)]
   |           -------------- `forbid` level set here
12 |
13 | #[allow(non_snake_case)] //~ ERROR E0453
   |         ^^^^^^^^^^^^^^ overruled by previous forbid
```

--- a/src/librustc/lint/context.rs
+++ b/src/librustc/lint/context.rs
@@ -606,11 +606,12 @@ pub trait LintContext: Sized {
                                                             "{}({}) overruled by outer forbid({})",
                                                             level.as_str(), lint_name,
                                                             lint_name);
+                    diag_builder.span_label(span, &format!("overruled by previous forbid"));
                     match now_source {
                         LintSource::Default => &mut diag_builder,
                         LintSource::Node(forbid_source_span) => {
-                            diag_builder.span_note(forbid_source_span,
-                                                   "`forbid` lint level set here")
+                            diag_builder.span_label(forbid_source_span,
+                                                    &format!("`forbid` level set here"))
                         },
                         LintSource::CommandLine => {
                             diag_builder.note("`forbid` lint level was set on command line")

--- a/src/test/compile-fail-fulldeps/lint-plugin-forbid-attrs.rs
+++ b/src/test/compile-fail-fulldeps/lint-plugin-forbid-attrs.rs
@@ -15,11 +15,13 @@
 #![plugin(lint_plugin_test)]
 #![forbid(test_lint)]
 //~^ NOTE lint level defined here
-//~| NOTE `forbid` lint level set here
+//~| NOTE `forbid` level set here
 
 fn lintme() { } //~ ERROR item is named 'lintme'
 
-#[allow(test_lint)] //~ ERROR allow(test_lint) overruled by outer forbid(test_lint)
+#[allow(test_lint)]
+//~^ ERROR allow(test_lint) overruled by outer forbid(test_lint)
+//~| NOTE overruled by previous forbid
 pub fn main() {
     lintme();
 }

--- a/src/test/compile-fail/E0453.rs
+++ b/src/test/compile-fail/E0453.rs
@@ -9,7 +9,10 @@
 // except according to those terms.
 
 #![forbid(non_snake_case)]
+//~^ NOTE `forbid` level set here
 
-#[allow(non_snake_case)] //~ ERROR E0453
+#[allow(non_snake_case)]
+//~^ ERROR allow(non_snake_case) overruled by outer forbid(non_snake_case)
+//~| NOTE overruled by previous forbid
 fn main() {
 }

--- a/src/test/compile-fail/lint-forbid-attr.rs
+++ b/src/test/compile-fail/lint-forbid-attr.rs
@@ -9,8 +9,10 @@
 // except according to those terms.
 
 #![forbid(deprecated)]
-//~^ NOTE `forbid` lint level set here
+//~^ NOTE `forbid` level set here
 
-#[allow(deprecated)] //~ ERROR allow(deprecated) overruled by outer forbid(deprecated)
+#[allow(deprecated)]
+//~^ ERROR allow(deprecated) overruled by outer forbid(deprecated)
+//~| NOTE overruled by previous forbid
 fn main() {
 }

